src/main/frontend/src/routes/persistence.svelte

```diff
@@ -30,6 +30,9 @@
 				{name: 'Not Applicable', value: 'NONE'},
 				{name: 'Event Based', value: 'EVENT_BASED'},
 			];
+			if($projectionsTypes.filter(p => p.value === projections).length < 1){
```

Comment:
```suggestion
			if(projections === 'OPERATION_BASED') {
```

Reply:
Yes this will work if "projectionsTypes" will not change in the future.